dataeng/jobs/analytics/SnowflakeMicrobachelorsITK.groovy

```diff
@@ -48,7 +49,7 @@ class SnowflakeMicrobachelorsITK {
                 }
             }
             triggers {
-                cron('0 6 * * 1-5')
+                cron('H 6 * * 1-5')
```

Comment:
Unrelated change but I noticed it when I was looking at the configuration in the UI:
![image](https://user-images.githubusercontent.com/3229735/78583016-386b9b80-7804-11ea-9a49-a8ba09b87df1.png)